src-ghc/Pact/GasModel/GasTests.hs

```diff
@@ -477,6 +478,19 @@ writeTests = defGasUnitTests allExprs
              ) |]
     allExprs = writeExpr :| []
 
+foldDBTests :: NativeDefName -> GasUnitTests
+foldDBTests = defGasUnitTests allExprs
+  where
+    writeExpr =
+      defPactExpression [text|
+      (let*
+        ((qry (lambda (k obj) true)) ;; select all rows
```

Comment:
Could it be interesting to see if there any gas costs associated with no rows being selected?

Reply:
should be the base gas cost.